### PR TITLE
[team_cppnoob_0]: testing pr

### PR DESCRIPTION
**Pull request by HaoyuanXu**
### What?

- brief: testing pr and write a helloworld program
- author: xhy
### What changes are included in this PR?

add a helloword program and a markdown file.
### Are these changes tested?
image

### CODE REVIEW

**Comment by Light-City** [comment]
pr title does not meet specifications

File: stl/team_cppnoob/xhy/main.exe
Comment (Light-City) [inline stl/team_cppnoob/xhy/main.exe:1]: should remove or add into .gitignore.

**Comment by Light-City** [comment]
The directory structure is incorrect, you can refer to what others have done, or this document

https://github.com/datawhalechina/whale-starry/blob/main/stl/rules/pr.md

File: stl/team_cppnoob/xhy/main.exe
Comment (HaoyuanXu) [reply stl/team_cppnoob/xhy/main.exe:1]: Thanks for correction! I have fixed them, may you check them again? Thanks for your time!

File: stl/team_cppnoob/xhy/src/main.cpp [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#include<iostream>
```
Comment (martinx) [inline stl/team_cppnoob/xhy/src/main.cpp:1]: 1  space is needed.

```cpp
#include<iostream>
```

=>

```cpp
#include <iostream>
```

File: stl/team_cppnoob/xhy/src/main.cpp [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+
+int main() {
+    std::cout << "hello world!" << std::endl;
```
Comment (martinx) [inline stl/team_cppnoob/xhy/src/main.cpp:4]: Indent 2 spaces

**Pull request closed without merging**